apps/main/src/i18n/locales/en/common.json

```diff
@@ -7,6 +7,7 @@
   "sell": "Sell",
   "buy": "Buy",
   "hide": "Hide",
+  "header.more": "More",
```

Comment:
I'd create a common translation key for that word, so it can be used somewhere else. "more": "More"